renderdoc/driver/gl/gl_debug.cpp

```diff
@@ -2249,8 +2249,8 @@ ResourceId GLReplay::RenderOverlay(ResourceId texid, CompType typeHint, DebugOve
     }
     else
     {
-      gl.glTextureImage2DEXT(DebugData.overlayTex, texBindingEnum, 0, eGL_RGBA16, texDetails.width,
-                             texDetails.height, 0, eGL_RGBA, eGL_FLOAT, NULL);
+      gl.glTextureImage2DEXT(DebugData.overlayTex, texBindingEnum, 0, eGL_RGBA16I, texDetails.width,
+                             texDetails.height, 0, eGL_RGBA_INTEGER, eGL_SHORT, NULL);
```

Comment:
Ah I see, I didn't realise baseline GLES3 didn't support render to FP16, I'd have thought that would only be missing in ES2.

I think your plan to do an extension check is the right direction. You could try falling back to just plain `GL_RGBA8` since I don't need values above 1.0, I just want higher precision. The triangle size and quad overdraw displays might start to get inaccurate without the precision of FP16, but I think it will mostly work.